packages/stream_video/lib/src/call/call_events.dart

```diff
@@ -263,6 +263,34 @@ class StreamCallCreatedEvent extends StreamCallEvent {
       ];
 }
 
+/// Event that is triggered when the call is created and another person is invited to participate.
+class StreamCallMissedEvent extends StreamCallEvent {
+  const StreamCallMissedEvent(
+    super.callCid, {
+    required this.sessionId,
+    required this.metadata,
+    required this.createdAt,
+    required this.callUser,
+    required this.members,
+  });
+
+  final String sessionId;
+  final CallMetadata metadata;
+  final DateTime createdAt;
+  final CallUser callUser;
+
+  /// List of members who missed the call
+  final List<CallMember> members;
+
+  @override
+  List<Object?> get props => [
+        ...super.props,
+        sessionId,
+        createdAt,
+        metadata,
+      ];
```

Comment:
Does props need to have all the other properties in the event?

Reply:
they should, added missing one 👍